DigitalLearningSolutions.Web/Views/LearningPortal/Current/_CurrentLearningResourceCard.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@model LearningResourceCardViewModel
+
+<div class="searchable-element nhsuk-panel @Model.DateStyle()" id="@Model.Id-lhr-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary nhsuk-u-padding-top-0 nhsuk-u-padding-left-0">
+      <span class="nhsuk-u-visually-hidden">@Model.DueByDescription()</span>
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.Id-name-lhr" name="name" data-name-for-sorting="name" role="heading" aria-level="2">
+        @Model.Name
+      </span>
+    </summary>
+    <div class="nhsuk-details__text nhsuk-u-padding-left-0">
+      <div class="nhsuk-u-padding-left-3">
+        <div class="nhsuk-grid-row">
+          <partial name="Shared/_TagRow" model="@Model" />
+        </div>
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-full">
+            @Html.Raw(Model.ResourceDescription)
+          </div>
+        </div>

```

Comment:
Looks like when this description is multiple lines, it runs uncomfortably close to the table below, might be nice to add some extra margin :)

Reply:
Partially increased the spacing. Thoughts?
![image](https://user-images.githubusercontent.com/80777689/144420652-e5a0a058-2935-46c9-9198-785ecf05e2e7.png)
<img src="https://user-images.githubusercontent.com/80777689/144420739-24a2525d-5b20-48a8-9349-52aa736ac4b6.png" width=320px />

Reply:
Just nitpicking, but I think it could use a tiny bit more margin. Upon close examination, it looks like the space above the description is a few pixels more than the space below the table. I think it would be good if they were the same, or even if the space below the description were a bit bigger because the text is less distinguishable from the table than the tags.

Reply:
Updated images with spacing equalised:

![Equalised spacing desktop](https://user-images.githubusercontent.com/80777689/144587309-5a86ab91-53ea-4934-94fe-c059c9520707.png)

<img src="https://user-images.githubusercontent.com/80777689/144587313-071b8d6a-ea9e-49b4-b9f7-ef3c8fd3f8bd.png" width=320px />

Reply:
Looks perfect

Reply:
Looks perfect!